test/runtests.jl

```diff
@@ -13,6 +13,17 @@ include("choosetests.jl")
 include("testenv.jl")
 include("buildkitetestjson.jl")
 
+longrunning_delay = if haskey(ENV, "JULIA_TEST_LONGRUNNING_DELAY") # minutes
+    parse(Int, ENV["JULIA_TEST_LONGRUNNING_DELAY"]) * 60
+else
+    45 * 60  # 45 minutes
+end
+longrunning_interval = if haskey(ENV, "JULIA_TEST_LONGRUNNING_INTERVAL") # minutes
+    parse(Int, ENV["JULIA_TEST_LONGRUNNING_INTERVAL"]) * 60
+else
+    15 * 60  # 15 minutes
+end
```

Comment:
Could use `get` instead of `if haskey` + `getindex`?

Reply:
I guess like this
```suggestion
longrunning_delay = parse(Int, get(ENV, "JULIA_TEST_LONGRUNNING_DELAY", "45")) * 60 # minutes
longrunning_interval = parse(Int, get(ENV, "JULIA_TEST_LONGRUNNING_INTERVAL", "15")) * 60 # minutes
```

Reply:
Globals can also be type-annotated with `::Int` not to be bare untyped _and_ non-const